Formula/neomutt.rb

```diff
@@ -7,6 +7,8 @@ class Neomutt < Formula
   option "with-lmdb", "Build with lmdb support"
   option "with-lua", "Build with lua scripting support enabled"
   option "with-s-lang", "Build against slang instead of ncurses"
+  option "with-fcntl", "Build with fcntl support"
```

Comment:
It might be worth adding a "(default)" comment here

Reply:
You mean like this?

```
option "with-fcntl", "Build with fcntl support (default)"
```

Reply:
Yes.  That's the preferred option and if the user doesn't specify anything, that's what they'll get.